sycl/test-e2e/Plugin/interop-level-zero-buffer.cpp

```diff
@@ -3,7 +3,7 @@
 // account direct calls to L0 API.
 // UNSUPPORTED: ze_debug
```

Comment:
This and same in other files needs to be removed (or rather changed to ur_l0_debug) as part of this PR.

Reply:
I can remove that.